c2/c2-client-bundle/c2-client-service/src/main/java/org/apache/nifi/c2/client/service/C2HeartbeatFactory.java

```diff
@@ -106,6 +106,17 @@ private AgentInfo getAgentInfo(AgentRepositories repos, RuntimeManifest manifest
         return agentInfo;
     }
 
+    private Set<SupportedOperation> getSupportedOperations(RuntimeManifest manifest) {
+        Set<SupportedOperation> supportedOperations;
+        // supported operations has value only in case of minifi, therefore we return empty collection if
```

Comment:
The RuntimeManifest is used by nifi-web module as well that's why I created a wrapper class. In case of NiFi this doesn't have any meaning for now, that's why I made this condition here.